### ISSUE TITLE
[Feature]: desktop support

### ISSUE THREAD

**Issue by laoshaw** [immich-app/immich#1427]
### Feature detail

do you have plan to add desktop support, if not could you?

### Platform

Server

**Comment by samip5** [comment]
What do you mean by desktop support?

**Comment by titro** [comment]
You mean as an extra App? Why not using it in the browser (as far as the web has the same functions as the mobile apps)?

**Comment by alextran1502** [comment]
If we are going to support desktop application, it will be under the form of an agent system, to watch a folder. We have no plan to add native viewer support at this time.

**Comment by Nebucatnetzer** [comment]
A sync agent for desktop would be great.

**Comment by cgonIT** [comment]
> A sync agent for desktop would be great.

Would be nice yes, to have a Windows desktop client to sync your photos easyly, as a backup for your account/server for example.

**Comment by elcilerdenbiri** [comment]
for macOS standards separate desktop experience should have ok

**Comment by riyaniii** [comment]
<img width="2644" height="1362" alt="Image" src="https://github.com/user-attachments/assets/77bd2746-94e4-471e-bc72-46286a316c7a" />

**Comment by lprimak** [comment]
Update: I was not able to find anything that correctly solves this problem, so I solved it myself.
I have created [Mirror-Immich](https://flowlogix.com/mirror-immich.html) - [GitHub Link](https://github.com/flowlogix/mirror-immich)
It creates exported symlink hierarchy, including all processed metadata that can be sync'd to Immich external library.
It also creates albums and syncs favorites optionally by using Immich API.
